velox/docs/monthly-updates/march-2023.rst

```diff
@@ -0,0 +1,75 @@
+********************
+March 2023 Update
+********************
+
+Documentation
+=============
+
+ * Add :doc:`Chapter 1  of the Programming Guide <../programming-guide/chapter01>`.
+ * Add blog post on `continuous integration and packaging. <https://velox-lib.io/blog/velox-build-experience>`_
+
+
+Core Library
+============
+
+ * Add `printVector` helper function.
+ * Add support for `copy_from` in GenericWriter.
+ * Add preliminary support for OffProcessExpressionEval operator.
+ * Add quota enforcement checks when allocating large chunks of memory.
+ * Deprecate `reallocate` Api from memory allocator.
+ * Optimize null aware anit-join when used with filter.
+ * Remove usage of CppToType.
+
```

Comment:
Also can mention that the aggregate function accumulator memory alignment is now fixed